gnovm/pkg/gnolang/debugger_test.go

```diff
@@ -23,49 +23,65 @@ type writeNopCloser struct{ io.Writer }
 
 func (writeNopCloser) Close() error { return nil }
 
-func eval(debugAddr, in, file string) (string, string, error) {
-	out := bytes.NewBufferString("")
-	err := bytes.NewBufferString("")
+// TODO (Marc): move evalTest to gnovm/tests package and remove code duplicates
```

Comment:
Note: I'm very happy if we want to keep `tests/` to contain test files only, while moving _how_ the tests are actually done into the relevant directories. To extend that idea, I kind of think that testscript (txtar) tests should go in `tests/`.

This is just to say, I think you can remove this error.